modules/TestUtils.js

```diff
@@ -88,6 +94,14 @@ export const functionThrows = (fn, context, args, value) => {
 export const arrayContains = (array, value, compareValues) =>
   array.some(item => compareValues(item, value) !== false)
 
+/**
+ * Returns true if the given array contains the value, false
+ * otherwise. The compareValues function must return false to
+ * indicate a non-match.
+ */
+export const setContains = (set, value, compareValues) =>
+  arrayContains([ ...set ], value, compareValues)
```

Comment:
it's a Set - why would we want to reify it as a real array, when it's got `.has()` which is O(1)?